assets/javascripts/issue_templates.js

```diff
@@ -376,13 +376,16 @@ ISSUE_TEMPLATE.prototype = {
         values = value
       }
 
+      let isChangedSelected = false;
       for (let i = 0; i < values.length; i++) {
         let options = document.querySelectorAll('#' + element.id + ' option')
         let filteredOptions = Array.from(options).filter(option => option.text === values[i])
         if (filteredOptions.length > 0) {
+          if (filteredOptions[0].selected == false) { isChangedSelected = true }
```

Comment:
Is this coming from the form as `false` or `"false"`? Also maybe better to use `===` instead of `==`.

Reply:
@zh 
Thank you for your feedback. ==== is correct.
`console.log(typeof(filteredOptions[0].selected)); # => boolean`

Reply:
Fixed by https://github.com/agileware-jp/redmine_issue_templates/pull/77/commits/b94d82fd2da564e8fb0e391cee64fb1aab647294 .